docs/install/Knative-with-Gloo.md

```diff
@@ -16,13 +22,11 @@ enabled. `kubectl` v1.10 is also required. This guide assumes that you've
 already created a Kubernetes cluster which you're comfortable installing _alpha_
 software on.
 
-This guide assumes you are using bash in a Mac or Linux environment; some
-commands will need to be adjusted for use in a Windows environment.
+This guide assumes you are using bash in a Mac or Linux environment; some commands will need to be adjusted for use in a Windows environment.
 
```

Comment:
@tcnghia  @ZhiminXiang is this correct? Are there other features that should be mentioned here?
```suggestion

- Some Knative features might not include support for Gloo. For example, the [automatic TLS certificate provisioning](../serving/using-auto-tls.md) is currently only supported in Istio.
```

Reply:
this is incorrect, Gloo supports the Automatic TLS feature (since it's included in the `Ingress` spec)